pkgs/development/tools/misc/binutils/default.nix

```diff
@@ -64,6 +64,12 @@ stdenv.mkDerivation rec {
 
     # https://sourceware.org/bugzilla/show_bug.cgi?id=22868
     ./gold-symbol-visibility.patch
+
+  ] ++ stdenv.lib.optionals targetPlatform.isAarch64 [
+    # Version 2.30 introduced strict requirements on ELF relocations which cannot
+    # be satisfied on aarch64 platform. Add backported fix from bugzilla.
+    # https://sourceware.org/bugzilla/show_bug.cgi?id=22764
+    ./relax-R_AARCH64_ABS32-R_AARCH64_ABS16-absolute.patch
```

Comment:
Using `fetchpatch { url = "https://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git;a=patch;h=279b2f94168ee91e02ccd070d27c983fc001fe12;hp=f3b91ed8f96a4d41eef1c64a7e77c406ec763612"; sha256 = "..."};` was not possible here because the patch was adapted?